player/command.c

```diff
@@ -3605,7 +3605,8 @@ static int mp_property_option_info(void *ctx, struct m_property *prop,
 
         union m_option_value def = m_option_value_default;
```

Comment:
could be null-initialized if it isn't meant to contain a valid value anyway

Reply:
It is, `m_option_value_default` is null opt basically.

Reply:
well either it's equivalent to `{0}` (then why does it exist?) or it's not.
but doesn't matter either way.